templates/download/index.html

```diff
@@ -0,0 +1,194 @@
+{% extends "templates/base.html" %}
+
+{% if title %}{% block title %} 系统下载 {{ title }}{% endblock %}{% endif %}
+
+{% if meta_description %}{% block meta_description %}Ubuntu 16.04下载, Ubuntu 18.04下载, Ubuntu 服务器下载， Ubuntu系统下载， Ubuntu官网系统下载， Ubuntu OpenStack{{ meta_description }}{% endblock %}{% endif %}
+
+{% block second_level_nav_items %}
+    {% include "templates/_nav_breadcrumb.html" with section_title="download" page_title="download"  %}
+{% endblock second_level_nav_items %}
+
+{% block content %}
+
+<div class="wrapper u-no-margin--top">
+<div id="main-content" class="inner-wrapper">
+
+<!-- main content -->
+<div class="p-strip is-deep is-bordered">
+	<div class="row">    
+      <h1>Ubuntu桌面下载</h1>  
+      <div class="col-12 p-card--highlighted">
```

Comment:
This should be is own row. Cols should always be direct decedents of a row.

Reply:
You mean h1 tag need to moved into Col12 ?